vpype_flow_imager/vpype_flow_imager.py

```diff
@@ -58,34 +67,34 @@
     "-ms",
     "--min_sep",
     default=0.8,
-    type=vpype_cli.FloatType(),
```

Comment:
What does the change from FloatType to LengthType mean?
Could it break anyones current workflow?
How is it useful to users?
Is the only difference that you can add an unit, like "mm" and it will recompute the value to pixels?

Reply:
All current values will be  accepted as-is, it's not a breaking change. But it will accept units and convert them to pixel values. This is useful to me to make sure that the "max size" and the "min separation" are both appropriate for the size of the page and the width of the pen.